cmd/activator/main.go

```diff
@@ -27,8 +27,9 @@ import (
 
 	"github.com/knative/pkg/configmap"
 	"github.com/knative/pkg/controller"
+	pkglogging "github.com/knative/pkg/logging"
```

Comment:
nit: Is the package renaming necessary here?

Reply:
We also import the serving logging package, so yes.

The one thing left that isn't in the shared logging package is the SyncFileWriter, otherwise we could drop the serving logging package import entirely.  cc @mdemirhan